webpack.common.config.js

```diff
@@ -36,16 +41,12 @@ var entry = {
     'profile-account-settings-page': staticPath('js/pages/profile-account-settings-page.js'),
     'profile-settings-applications-list-page': staticPath('js/pages/profile-settings-applications-list-page.js'),
     'profile-settings-applications-detail-page': staticPath('js/pages/profile-settings-applications-detail-page.js'),
-    'register_1-page': staticPath('js/pages/register_1-page.js'),
     'sharing-page': staticPath('js/pages/sharing-page.js'),
     'conference-page': staticPath('js/pages/conference-page.js'),
     'meetings-page': staticPath('js/pages/meetings-page.js'),
     'view-file-tree-page': staticPath('js/pages/view-file-tree-page.js'),
-    'project-settings-page': staticPath('js/pages/project-settings-page.js'),
     'search-page': staticPath('js/pages/search-page.js'),
-    'registration-retraction-page': staticPath('js/pages/registration-retraction-page.js'),
     'share-search-page': staticPath('js/pages/share-search-page.js'),
-    'profile-settings-addons-page': staticPath('js/pages/profile-settings-addons-page.js'),
```

Comment:
You may be getting the old file from before it was removed... Once I cleaned prebuilt assets, the file was not rebuilt, and I started getting 500 errors for loading the file.